test/cbmc/proofs/MakefileCommon.json

```diff
@@ -32,9 +32,7 @@
 
     "CBMCFLAGS ": [
 	"--object-bits 8",
-	"--32",
-	"--bounds-check",
-	"--pointer-check"
+	"--32"
     ],
```

Comment:
Are these options (`bounds-check`, `pointer-check`) not required now?

Reply:
@tautschnig Could you help answer this question?

Reply:
With CBMC v6 those are enabled by default (any checks that assess undefined behaviour per the C standard are now enabled by default).